styles.css

```diff
@@ -475,6 +476,151 @@ button::selection {
   background-color: #5e42d7;
 }
 
+/* Pop-up window styling starts here */
+
+.pop-up-window {
```

Comment:
- [ ] The popup should fit the window size , in your design it does not.
- [ ] The popup should be the only element fixed on the surface. The rest of the website should be scrollable and blurred in the background. Accessible only when the user exits the popup. In your design the rest of the website is accessible , the user can even open multiple popups in the webpage. Hence it is not behaving like a popup, instead it is like another section added to the webpage.

Kindly fix these issues
Cheers 👍 

## Your Design
![William Raga's Portfolio](https://user-images.githubusercontent.com/60618877/179916174-9333c45b-2196-40b4-a30e-ab94a3a2f1cd.gif)